src/domains/creator/studio-quality-inspection.ts

```
@@ -0,0 +1,1573 @@
+    });
+  }
+}
+
+function inspectTextCharacters(
+  page: PageState,
+  pageIndex: number,
+  el: Extract<El, { type: "text" | "bubble" | "sticker" }>,
+  add: (issue: MutableIssue) => void
+): void {
+  const text = el.text ?? "";
+  if (/[\u0000\uFFFD]/u.test(text)) {
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00.

(no-control-regex)